### PR TITLE
really really final it is re~~~~~~~~~~~~al!!!

### PR DESCRIPTION
**Pull request by kanghyejong**
confirm please it is really final it is perpect

### CODE REVIEW

**Comment by ywnam-stephen** [comment]
much obliged

**Comment by haeramkeem** [comment]
hot doggo brother

**Comment by ywnam-stephen** [comment]
Ddang ddang ddang

File: src/main/java/org/cnu/realcoding/service/DogManagementService.java [hunk 1]
```diff
@@ -54,33 +41,56 @@ public void insertDog(Dog dog) {
...
-        } else {
-            dogRepository.updateDogKind(name, ownerName, ownerPhoneNumber, changeKind);
         }
     }
 
+    // allkey 조회
     public Dog getDogByAllKey(String name, String ownerName, String ownerPhoneNumber){
         Dog mongo_dog = dogRepository.getDogByAllKey(name, ownerName, ownerPhoneNumber);
-        if(!dogRepository.check_Exist(name, ownerName, ownerPhoneNumber)) {
+        if(dogRepository.check_Exist(name, ownerName, ownerPhoneNumber)) {
```
Comment (ywnam-stephen) [inline src/main/java/org/cnu/realcoding/service/DogManagementService.java:63]: What have you done

**Review by ywnam-stephen — COMMENTED** [review]
I am error.

File: src/main/java/org/cnu/realcoding/service/DogManagementService.java (hunk 1 above)
Comment (songyoungwoon) [reply src/main/java/org/cnu/realcoding/service/DogManagementService.java:63]: no way

**Pull request merged**